types/account.ts

```diff
@@ -319,12 +321,7 @@ export interface AccountLimits {
             }
         }
         card: {
-            limits: {
-                dailyWithdrawal: number
-                dailyDeposit: number
-                dailyPurchase: number
-                dailyCardTransaction: number
-            }
+            limits: CardLevelLimits
```

Comment:
@ilyamerman should we be replacing limits set at the deposit product level with the card-level limits? I think we want to maintain account-level limits